src/features/content-explorer/content-explorer-modal/ContentExplorerModal.js

```diff
@@ -12,10 +12,14 @@ import './ContentExplorerModal.scss';
 
 type Props = {
     additionalColumns?: Array<React.ComponentType<Column>>,
+    /** Items that will show up as selected */
+    autoSelectedItems: Object,
     breadcrumbProps?: BreadcrumbProps,
     className?: string,
     customInput?: React.ComponentType<any>,
     description?: string,
+    // Called when items are deselected
+    deselectAutoSelectedItem: Function,
```

Comment:
nit: the comments aren't included in the first. my guess because the container already has the descriptions for them